src/Test/Perf/infra/monitor_and_start_test.csx

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+Task.Run(() => StartWatching());
+Console.WriteLine("Press any key to continue");
+Console.Read();
+
+void StartWatching()
+{
+    FileSystemWatcher watcher = new FileSystemWatcher();
+    watcher.Path = Args[0];
+    watcher.Created += new FileSystemEventHandler(OnChanged);
+    watcher.IncludeSubdirectories = false;
+    watcher.EnableRaisingEvents = true;
+    while (true)
```

Comment:
Aren't the signed builds in MicroBuild? Can't you just have a TFS build agent do this? (like we used to)